src/main.go

```diff
@@ -8,113 +8,36 @@ import (
 
 func main(){
 	
-	// test food items to test user fxns
-	testFoodItem := BackendPkg.FoodItem{
-		Name:        "peanut butter",
-		StoreCost:   369.99,
-		OnSale:      true,
-		SaleDetails: "BOGO",
-		Quantity:    10,
-	}
```

Comment:
Nice removing these, are they no longer needed anywhere? thoughts on getting some hardcoded json values to store our data in case the .db file breaks?